fe/fe-core/src/main/java/com/starrocks/credential/hdfs/HDFSCloudConfigurationFactory.java

```diff
@@ -0,0 +1,62 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.credential.hdfs;
+
+import autovalue.shaded.com.google.common.common.base.Preconditions;
+import com.starrocks.credential.CloudConfiguration;
+import com.starrocks.credential.CloudConfigurationFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.starrocks.credential.CloudConfigurationConstants.HDFS_AUTHENTICATION;
+import static com.starrocks.credential.CloudConfigurationConstants.HDFS_KERBEROS_KEYTAB;
+import static com.starrocks.credential.CloudConfigurationConstants.HDFS_KERBEROS_KEYTAB_CONTENT;
+import static com.starrocks.credential.CloudConfigurationConstants.HDFS_KERBEROS_PRINCIPAL;
+import static com.starrocks.credential.CloudConfigurationConstants.HDFS_PASSWORD;
+import static com.starrocks.credential.CloudConfigurationConstants.HDFS_USER_NAME;
+
+public class HDFSCloudConfigurationFactory extends CloudConfigurationFactory {
+    private final Map<String, String> properties;
+
+    public HDFSCloudConfigurationFactory(Map<String, String> properties) {
+        this.properties = properties;
+    }
+
+    @Override
+    protected CloudConfiguration buildForStorage() {
+        Preconditions.checkNotNull(properties);
+        Map<String, String> haConfigurations = new HashMap<>(properties);
+        haConfigurations.remove(HDFS_AUTHENTICATION);
```

Comment:
I think there is no need to remove?

Reply:
> I think there is no need to remove?

haConfigurations refer to https://docs.starrocks.io/zh-cn/latest/sql-reference/sql-statements/data-manipulation/BROKER%20LOAD#hdfs